tests/resources/Util.robot

```diff
@@ -177,6 +177,8 @@ Run VIC Machine Delete Command
     Should Be Equal As Integers  ${rc}  0
     Should Contain  ${output}  Completed successfully
     [Return]  ${output}
+    Run  rm -f ${vch-name}/*
```

Comment:
I was talking about putting the `rm` commands before the `[Return]`. Anyway, I checked; turns out `[Return]` doesn't necessarily have to be the last command in the keyword.